modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -442,30 +442,28 @@
                         "example": "Compensation"
                       },
                       "claimDate": {
-                        "type": [
-                          "string",
-                          "null"
-                        ],
                         "format": "date",
+                        "type": "date",
+                        "nullable": true,
                         "description": "Date in YYYY-MM-DD the claim was first filed",
                         "example": "2018-06-04"
                       },
                       "claimPhaseDates": {
                         "type": "object",
                         "properties": {
                           "phaseChangeDate": {
+                            "format": "date",
                             "type": "date",
```

Comment:
looks like these will actually change to string per [this](https://github.com/department-of-veterans-affairs/vets-api/pull/11574/files#r1083168127).